org.hdfgroup.HDFView.yml

```diff
@@ -51,11 +52,18 @@ modules:
       - -DBUILD_TESTING:BOOL=OFF
       - -DBUILD_SHARED_LIBS:BOOL=ON
       - -DBUILD_STATIC_LIBS:BOOL=OFF
+      - -DJava_JAVA_EXECUTABLE=/usr/lib/sdk/openjdk/bin/java
+      - -DJava_JAR_EXECUTABLE=/usr/lib/sdk/openjdk/bin/jar
+      - -DJava_JAVAC_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javac
+      - -DJava_JAVAH_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javah
+      - -DJava_JAVADOC_EXECUTABLE=/usr/lib/sdk/openjdk/bin/javadoc
+      - -DJAVA_INCLUDE_PATH=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/
+      - -DJAVA_INCLUDE_PATH2=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/linux/
+      - -DJAVA_AWT_INCLUDE_PATH=/usr/lib/sdk/openjdk/jvm/openjdk-22/include/
     sources:
       - type: archive
-        url: https://support.hdfgroup.org/ftp/HDF/releases/HDF4.2.16-2/src/hdf-4.2.16-2.tar.bz2
-        sha256: c5c3234b5012258aef2e4432f649b31c21b26015afba1857ad83640c3f2b692c
-        strip-components: 2
+        url: https://github.com/HDFGroup/hdf4/archive/refs/tags/hdf4.3.0.tar.gz
+        sha256: a6639a556650e6ea8632a17b8188a69de844bdff54ce121a1fd5b92c8dd06cb1
```

Comment:
Hummm, according to [HDF 4.3.0 Release Information](https://support.hdfgroup.org/downloads/hdf4/hdf4_3_0.html), the SHA256 for [hdf4.3.0.tar.gz](https://github.com/HDFGroup/hdf4/archive/refs/tags/hdf4.3.0.tar.gz) should be `282b244a819790590950f772095abcaeef405b0f17d2ee1eb5039da698cf938b`.  I guess we should prefer [official tarballs](https://support.hdfgroup.org/releases/hdf4/v4_3/v4_3_0/downloads/index.html) here: [hdf5-1.14.5.tar.gz](https://support.hdfgroup.org/releases/hdf5/v1_14/v1_14_5/downloads/hdf5-1.14.5.tar.gz)